client/a8c-for-agencies/components/a4a-feedback/lib/get-a4a-feedback-props.ts

```diff
@@ -0,0 +1,53 @@
+import { FeedbackType } from '../types';
+
+type A4AfeedbackProps = {
+	title: string;
+	description: string;
+	questionDetails: string;
+	ctaText: string;
+};
+
+export const getA4AfeedbackProps = (
+	type: FeedbackType,
+	translate: ( key: string, args?: Record< string, unknown > ) => string,
+	args?: Record< string, unknown >
+): A4AfeedbackProps => {
+	switch ( type ) {
+		case 'referral-complete':
+			return {
+				title: translate( 'Your referral order is complete!' ),
+				description: translate(
+					'Your referral order was emailed to %(email)s for payment. Once they pay, you can assign the products to a site.',
+					{ args: { email: args?.email } }
+				) as string,
+				questionDetails: translate( 'How was your experience making a referral?' ),
+				ctaText: translate( 'Submit and continue to Dashboard' ),
+			};
+		case 'agency-details-added':
+			return {
+				title: translate( 'Agency details added!' ),
+				description: translate(
+					"Nice job! Your information has been added to the your agency's public profile."
```

Comment:
Small typo: This should be either "the" or "your," not "**the your**".

Reply:
Thanks for reporting. Fixed it here: https://github.com/Automattic/wp-calypso/pull/97397